ignite/metrics/gan/utils.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, return_features: bool, device: Union[str, torch.device] = "cp
             raise RuntimeError("This module requires torchvision to be installed.")
         super(InceptionModel, self).__init__()
         self._device = device
+        self.return_features = return_features
```

Comment:
Why do we need this as an attribute ?

Reply:
For the if else condition in the forward function

Reply:
We can append softmax layer once in the model definition.

Reply:
We can't just append to a sequential. We will have to create a new one.
If you are fine with that I will change the implementation that way.